default/python/AI/ColonisationAI.py

```diff
@@ -654,6 +660,8 @@ def evaluate_planet(planet_id, mission_type, spec_name, detail=None):
     capital_id = PlanetUtilsAI.get_capital()
     homeworld = universe.getPlanet(capital_id)
     planet = universe.getPlanet(planet_id)
+    prospective_invasion_targets = [pid for pid, pscore, trp in
+                                    AIstate.invasionTargets[:PriorityAI.allottedInvasionTargets] if pscore > 20]
```

Comment:
The value of ```allottedInvasionTargets``` is currently updated after colonization planning so it would always use last turn's value which would be zero in case of a loaded game...

It is defined as ```allottedInvasionTargets = 1 + int(fo.currentTurn() / 25)``` which is independent from other calculations, so it should probably be redefined as a function in an appropriate module which solves the timing issue (and possibly resolves some minor bugs in existing code as well...)
```
def allotted_invasion_targets():
    return 1 + int(fo.currentTurn() / 25)
```

Reply:
yes, good suggestion